scripts/llvm-arw

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# shellcheck disable=SC2068
```

Comment:
What is this disabling the warning for? It's not attached to any code.

Reply:
Fixes the shell check errors in the function below